opentech/apply/funds/views.py

```diff
@@ -90,6 +90,33 @@ def form_valid(self, form):
         return super().form_valid(form)
 
 
+@method_decorator(staff_required, name='dispatch')
+class ScreeningSubmissionView(DelegatedViewMixin, UpdateView):
+    model = ApplicationSubmission
+    form_class = ScreeningSubmissionForm
+    context_name = 'screening_form'
+
+    def form_valid(self, form):
+        action = form.cleaned_data.get('action')
```

Comment:
If you are making the changes above to the form then this simplifies as per the `UpdateLeadView` (i think).


1. get old value before save - no need to do the copy as you should have the screening object not the submission instance
2. save using super
3. send message

I would rather the messaging handles any of the logic around what is shown (i.e. set/set from), so just passing None or the object is the way to go.

Reply:
You might have some fun with the generic relationship on the activity, but will address that in #806

Reply:
Yes, this all makes total sense! The messaging was a little tricky, but then I realized I could send in a single character if screening status is None ("-") so that the messaging would make sense and not error out on getting a None value.

Reply:
OK, lets just confirm when looking at #806 that this still makes sense.